components/telemetry-operator/main.go

```diff
@@ -148,7 +148,7 @@ func main() {
 	}
 
 	restartsTotal := prometheus.NewCounter(prometheus.CounterOpts{
-		Name: "telemetry_fluentbit_restarts_total",
+		Name: "telemetry_fluentbit_triggered_restarts_total",
```

Comment:
There should also be a ServerMonitor that might need a change.

Reply:
Fixed :)